storage/Dockerfile

```diff
@@ -11,7 +11,7 @@ RUN git clone https://github.com/spdk/spdk --branch ${TAG} --depth 1 && \
     cd spdk && git submodule update --init --depth 1 && scripts/pkgdep.sh --rdma
 
 # hadolint ignore=DL3003
-RUN cd spdk && ./rpmbuild/rpm.sh --without-uring --without-crypto \
+RUN cd spdk && CFLAGS=-mno-avx512f ./rpmbuild/rpm.sh --without-uring --without-crypto \
```

Comment:
I think we should add an ARG to control this behavior.  It can default to no AVX.

Reply:
agree